docs/cardinal/game/configuration/nakama.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Nakama'
+description: 'This section configures Nakama, including allowlist, metrics, and tracing settings.'
+---
+
+```
+[nakama]
+ENABLE_ALLOWLIST = boolean
+OUTGOING_QUEUE_SIZE = integer
+NAKAMA_TRACE_ENABLED = boolean
+NAKAMA_METRICS_ENABLED = boolean
+```
+
+### ENABLE_ALLOWLIST
+
+Enables Nakama’s allowlist feature, which allows for beta keys. This is useful for restricted access to certain parts of the game during development or early access.
+
+**Importance** : Set this to true if you plan to use Nakama’s beta key system.
+
+**Example**
+```
+ENABLE_ALLOWLIST = 'false'
```

Comment:
Some values in the starter game template's `world.toml` file contains string booleans (`'true'`, `'false'`). We should convert this all to boolean. Not sure if it'll affect the behavior of the system though. Need to check.

Reply:
We can write it as boolean because in the world engine it will parsed to boolean value.

But if you are running cardinal using world cli, you can use both boolean or string because world cli treat all the value as string.